src/applications/vaos/components/BackLink.jsx

```diff
@@ -14,53 +15,30 @@ export default function BackLink({ appointment }) {
   } = appointment.vaos;
 
   const location = useLocation();
+  const history = useHistory();
   const dispatch = useDispatch();
 
-  const handleClickGATracker = () => {
-    let status;
-    if (isPendingAppointment) {
-      status = 'pending';
-    } else if (isUpcomingAppointment) {
-      status = 'upcoming';
-    } else if (isPastAppointment) {
-      status = 'past';
-    }
+  let status;
+  let link = '/';
+  let linkText = 'Back to appointments';
+  if (isPendingAppointment) {
+    status = 'pending';
+    link = '/pending';
+    linkText = 'Back to request for appointments';
+  } else if (isUpcomingAppointment) {
+    status = 'upcoming';
+  } else if (isPastAppointment) {
+    status = 'past';
+    link = '/past';
+    linkText = 'Back to past appointments';
+  }
 
-    const progress =
-      location.search === '?confirmMsg=true' ? 'confirmation' : 'appointment';
+  const progress =
+    location.search === '?confirmMsg=true' ? 'confirmation' : 'appointment';
 
-    if (progress === 'confirmation' && status === 'upcoming') {
-      status = 'direct';
-    }
-    return () => {
-      recordEvent({
-        event: `${GA_PREFIX}-${status}-${progress}-details-descriptive-back-link`,
-      });
-
-      dispatch(closeCancelAppointment());
-    };
-  };
-
-  const handleBackLinkText = () => {
-    // Default to upcoming page
-    let linkText = 'Back to appointments';
-    if (isPendingAppointment) {
-      linkText = 'Back to request for appointments';
-    } else if (isPastAppointment) {
-      linkText = 'Back to past appointments';
-    }
-    return linkText;
-  };
-  const handleBackLink = () => {
-    // Default to upcoming page
-    let link = '/';
-    if (isPendingAppointment) {
-      link = '/pending';
-    } else if (isPastAppointment) {
-      link = '/past';
-    }
-    return link;
-  };
+  if (location.search === '?confirmMsg=true') {
```

Comment:
Hmm this looks like it's new logic, what's the reason for needing this?

Reply:
Good catch. It's superfluous so removing this logic